ocaml_typing/402/location.ml

```diff
@@ -104,24 +104,24 @@ let highlight_terminfo ppf num_lines lb locs =
   (* If too many lines, give up *)
   if !lines >= num_lines - 2 then raise Exit;
   (* Move cursor up that number of lines *)
-  flush stdout; Terminfo.backup !lines;
+  flush stdout; Terminfo.backup stdout !lines;
```

Comment:
It would probably be good to have a set of patch files, keeping track of what changes we've made to the original ocaml source

Reply:
(don't need to do that for this PR though)